src/main/java/duke/commands/AddDeadlineCommand.java

```diff
@@ -0,0 +1,57 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.ItemList;
+import duke.items.Deadline;
+import duke.Ui;
+import java.util.ArrayList;

```

Comment:
Wrong Import statement order and 
duke.Ui should be before duke.items.Deadline

```suggestion
import java.util.ArrayList;

import duke.DukeException;
import duke.ItemList;
import duke.Ui;
import duke.items.Deadline;
```